biojava-structure/src/main/java/org/biojava/nbio/structure/cluster/SubunitExtractor.java

```diff
@@ -75,7 +75,7 @@ private SubunitExtractor() {
 				logger.debug("Chain " + c.getId() + "; CA Atoms: " + ca.length + "; SEQRES: " + c.getSeqResSequence());
 				if (ca.length==0)
 					continue;
-				subunits.add(new Subunit(ca, c.getName(), null, structure));
+				subunits.add(new Subunit(ca, c.getId(), null, structure));
```

Comment:
Could you, please, also correct docs to reflect this change?

Reply:
Good catch. Just pushed a commit